h2o-core/src/main/java/water/api/schemas3/ModelSchemaV3.java

```diff
@@ -43,6 +43,8 @@
   @API(help="Checksum for all the things that go into building the Model.", direction=API.Direction.OUTPUT)
   protected long checksum;
 
+  private PS input_parameters;
```

Comment:
@koniecsveta Why is this field private-only? It is surprising to see something like that in a schema class.

Reply:
It's due to the unique way parameters are serialized to JSON in this class: so this private field is used to transfer the input value from the model instance to the final JSON.

cf. also conversation in `SchemaMetadata`

personally, I don't have any problem with private fields in schemas in that case.
I'm much more confused by private fields exposed as API in `TimelineV3`, which looks like nonsense to me.